### ISSUE TITLE
How to listing Fungible Token in NFT Marketplace?

### ISSUE THREAD

**Issue by yodhaanoraga** [MeshJS/mesh#163]
Hi MeshJs Dev Team,

I am wondering how I can do modification the basic feature NFT marketplace for support selling Fungible Token?
Because in my case, I need to allow user to sell their fungible token (quantity more than 1). And as I can see in the smart contract, its only can set price data.

So, I have read in the listing method is like this
`const txList = await marketplace.listAsset(
          walletAddress,
          detail.unit,
          parseInt(listPrice) * 1000000,
         quantity  // <<------- how to customize to add this field ?
        );
        const res = await addListing({
          ...detail,
          listing: {
            seller: walletAddress,
            price: parseInt(listPrice) * 1000000,
          },
        });`

Basically, why I need to allow sell FT in NFT marketplace? because I thought this FT is consider as the online ticketing for artist concert. Please let me know if MeshJs Team have another simple solution for accommodate my requirement so I don't need to heavy customize in existing code.

Thank you in advance.

**Comment by HinsonSIDAN** [comment]
Marketplace contract has been refactoring to aiken. In general these contracts could not be easily customized by only importing from `@meshsdk/contract`. If you want custom logic, everything is open source and you can add your tweak anywhere. Please reach out if you still got any issue
